htdocs/stripe/class/stripe.class.php

```diff
@@ -333,6 +333,7 @@ public function getPaymentIntent($amount, $currency_code, $tag, $description = '
 		dol_syslog("getPaymentIntent", LOG_INFO, 1);
 
 		$error = 0;
+		$description .= ' TIME : '.time();
```

Comment:
It means we can't use anymore the noidempotency_key feature.
If you want to avoid payment due to same request submited twice with same description, you must set the field noidempotency_key to 1.

Reply:
ok thanks, but it means in v12 there's no possibility to use noidempotency_key : parameter exists but isnt editable by user

Reply:
This is not a parameter for user but a parameter for the code that will call API.
I think we should fix v12 by backporting the same change of default parameter than the one done in v13.0.1

Reply:
ok thanks